po/README.md

```diff
@@ -0,0 +1,32 @@
+i18n
+====
+
+This directory contains the translations of Piper.
+
+For errors in translations, please [file an
+issue](https://github.com/libratbag/piper/issues/new).
+
+New or updated translations are always welcome. To start a new translation, run:
+
+	$ ninja -C $builddir piper-pot
+	$ cp po/piper.pot po/$lang.po
```

Comment:
two things here: make it with a meson command so you can use less variables like `$builddir`. And the `cp` won't work because it's in the builddir. Make it something like:

````
  $ meson translation-build
  $ ninja -C translation-build piper-pot
  $ cp translation-build/po/piper.pot po/$lang.po
  # you can remove the build directory now
  $ rm -rf translation-build
  # add the new file to git
  $ git add po/$lang.po
  # now edit the file and add translations
````